### DISCUSSION TITLE
locking button

### DISCUSSION THREAD

**Discussion by Gildas224** [ScratMan/HASmartThermostat#229] [Ideas]
Hello,
Is it possible to add a locking button, if it's enabled, we or HA couldn't modify the température, and if it disable we can do as we want?

**Comment by Chupaka** [comment]
What's the use case?

**Reply by Gildas224** [reply]
For example if you are in holliday at home or if you have a guest who comes unexpectedly.

**Reply by Chupaka** [reply]
You mean, your guest has access to your Home Assistant, but you need to forbid to make target temperature changes for him?..

**Reply by Gildas224** [reply]
My guest hasn't access to my home assistant, but if you have a program for your guest room (example: you are heating it when electricity is cheaper, or you have sensitive to the cold guest and don't want to change the value during the night, or your room is under renovation) for example, but you don't want to use program.

**Reply by ScratMan** [reply]
Add a button card next to your thermostat card to control your scheduler/automation that manages the setpoint.
I can't add a button on the thermostat, it's in the frontend.
